css/stanford-benefits.css

```diff
@@ -61,6 +61,14 @@ body.not-logged-in #edit-field-su-plan-year-value-wrapper {
 
 .view-caw-benefit-comparison-tool article .field-group-fieldset {
   margin-top: 40px; }
+  .view-caw-benefit-comparison-tool article .field-group-fieldset.hide {
```

Comment:
Weird indentation. Why are these being hidden with CSS rather than removed from the View?